XRTK-Core/Packages/com.xrtk.core/Runtime/Providers/Controllers/BaseController.cs

```diff
@@ -252,23 +233,10 @@ public async Task TryRenderControllerModelAsync(byte[] glbData = null, bool useA
                     ? cameraSystem.MainCameraRig.PlayspaceTransform
                     : CameraCache.Main.transform.parent;
 
-                // If the model was loaded from a system template
-                if (gltfObject != null)
```

Comment:
We need to provide an alternative

Reply:
The alternative is ultimately to use models in your project obviously.

Another way is to have in the sdk have a feature that can generate the model and then set the model parameter using a gender gltf interface.

But without a gltf system built into the core, the core can't enable that. But tbh most projects don't use this

Reply:
I'm using the gltf to load controller models for wmr and oculus

Reply:
So what is your suggestion to address this if you are loading models?  Given the aim is to allow users to use their preferred glTF importer.

Reply:
Can we at least provide a game object to instantiate?

Reply:
OK, doing some testing and the simplest way would be for the developer to update the visualizer type for the controller to be one to load the glTF model required.

You don't need a GameObject to attach to, simply use your own visualizer that loads your glTF model.

Thoughts @StephenHodgson

Reply:
Where is `gltfObject` declared? Shouldn't we remove that as well?

Reply:
> Where is `gltfObject` declared? Shouldn't we remove that as well?

Line 222, it was removed.  THere is no longer any glTF reference in the Core classes post this change